src/models/Filter.js

```diff
@@ -138,6 +138,13 @@ export default class Filter {
         if (!orCondition) {
           orCondition = true;
           where = [];
+          // Incase only 1 verify status selected, then the above statement overwrites the previous "where" status.
```

Comment:
This method is starting to become quite difficult to follow and easy to break.
LoopBack supports passing a single item to an `or` statement. Perhaps we should _always_ assume we will use `or` for `verifyStatus` and `planterId`, regardless of the number of values.

The database should optimise the query anyway, so I don't think it will have a negative impact on performance.

Reply:
Got it. With that assumption, there is no need for an "orCondition" flag as it Is always going to be true.
I will work on that. Will the new commit automatically be added to this PR?

Reply:
Yes, just push additional commits and the PR will be updated.

Reply:
@nmcharlton , I have pushed the new commit. Requesting your review for the same.